.github/workflows/release-please.yml

```
@@ -61,3 +61,17 @@ jobs:
     permissions:
       contents: read
       id-token: write
+
+  # release-please outputs come from conventional-commit aggregation, not from
+  # untrusted issue/PR event payloads, so direct ${{ ... }} interpolation is
+  # safe here. Same pattern as the publish jobs above.
+  publish-wren-langchain:
+    needs: release-please
+    if: needs.release-please.outputs['wren-langchain--release_created'] == 'true'
```

actionlint (1.7.12)

[error] property "wren-langchain--release_created" is not defined in object type {wren--release_created: string; wren--tag_name: string; wren--version: string; wren-core-py--release_created: string; wren-core-py--tag_name: string; wren-core-py--version: string; wren-core-wasm--release_created: string; wren-core-wasm--tag_name: string; wren-core-wasm--version: string}

(expression)